lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
+    <i className="fa fa-circle" style={{ color: props.color }}></i> { props.label }
```

Comment:
If this will just be a coloured circle I don't think we need `fa`. Plus we'll move away from that at some point (not super keen on font icons).

How about using a BEM element class e.g. `status-indicator__circle`?